### PR TITLE
Use PascalCase for preact component modules  and related assets

### PR DESCRIPTION
**Pull request by lyzadanger**
This PR makes the following changes:

* Make sidebar preact component modules (filenames) use PascalCase
* Make tests for these preact component modules use PascalCase (e.g. `AnnotationBody-test.js`)
* Make corresponding SCSS modules PascalCase
* Update imports and CSS class names to use Pascal-Cased variants

Annotator components haven't been touched (yet).

### CODE REVIEW

**Comment by lyzadanger** [comment]
Putting this in draft as I work through fixing CI issues: pitfalls of working on OS that isn't truly case-sensitive.

**Comment by lyzadanger** [comment]
codecov fail is a head-scratcher...

**Comment by lyzadanger** [comment]
That testcov fail ain't my doing.

File: src/sidebar/components/HelpPanel.js [hunk 1]
```diff
@@ -25,17 +25,17 @@ import VersionInfo from './version-info';
  */
 function HelpPanelTab({ linkText, url }) {
   return (
-    <div className="help-panel-tabs__tab">
+    <div className="HelpPanel-tabs__tab">
```
Comment (LMS007) [inline src/sidebar/components/HelpPanel.js:28]: This one looks off.

Should be this `HelpPanelTab__tab` or perhaps just `HelpPanelTab`

Comment (LMS007) [inline src/sidebar/components/HelpPanel.js:28]: This one looks off.

Should be this `HelpPanelTab__tab` or perhaps just `HelpPanelTab`

File: src/sidebar/components/Spinner.js [hunk 2]
```diff
@@ -4,13 +4,13 @@ import { createElement } from 'preact';
  * Loading indicator.
  */
 export default function Spinner() {
-  // The `spinner__container` div only exists to center the spinner within
-  // the `<spinner>` Angular component element. Once consumers of this component
```
Comment (LMS007) [inline src/sidebar/components/Spinner.js:8]: hah, good catch

**Review by LMS007 — APPROVED** [review]
I just found one questionable class name that I think should change then this looks GTG. Also we will need to switch out components in frontend-shared/src perhaps with or after the annotator changes. Currently there is no /components folder in frontend-shared but that maybe should change.

**Comment by robertknight** [comment]
I have rebased this on master after the change to move `SvgIcon` into the frontend-shared package.

File: src/sidebar/components/AutocompleteList.js [hunk 3]
```diff
@@ -53,14 +53,14 @@ export default function AutocompleteList({
         // The parent <input> field should capture keyboard events
         // eslint-disable-next-line jsx-a11y/click-events-have-key-events
         <li
-          key={`autocomplete-list-${index}`}
+          key={`AutocompleteList-${index}`}
```
Comment (robertknight) [inline src/sidebar/components/AutocompleteList.js:56]: The key could just be `index` here, since it only needs to be unique amongst its siblings.

**Review by robertknight — APPROVED** [review]
LGTM.

**Pull request merged**